boards/arm/sam4s_xplained/ext_ram.c

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (c) Justin Watson 2017
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <device.h>
+#include <init.h>
+#include <nanokernel.h>
+#include <pinmux.h>
+#include <soc.h>
+#include <sys_io.h>
+
+/* Static Memory Controller (SMC) */
```

Comment:
It wouldn't be all that bad to have support for an external RAM. But being the first one is always tricky.